### DISCUSSION TITLE
posts/install-gitlab-with-docker-on-ubuntu-18-04/

### DISCUSSION THREAD

**Discussion by giscus** [linhongzhou/linhongzhou.github.io#1] [General]
# posts/install-gitlab-with-docker-on-ubuntu-18-04/

In this tutorial, you will install and configure GitLab, a popular open-source application that primarily hosts Git repositories, on Ubuntu 18.04.

http://localhost:4000/posts/install-gitlab-with-docker-on-ubuntu-18-04/

**Comment by linhongzhou** [comment]
test
